posthog/migrations/0147_event_props_fuzzy_search.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.1.8 on 2021-04-27 01:13
+
+import django.contrib.postgres.indexes
+from django.contrib.postgres import operations
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0146_eventproperty_sync"),
+    ]
+
+    operations = [
+        operations.TrigramExtension(),
```

Comment:
Need confirmation from deployments team that it's okay to run this migration. I'm not sure if we're too limited with the superuser permissions requirement. CC @fuziontech @macobo

Reply:
I'm very unconvinced we'll be able to run this successfully on all our various deployments. Instead of fuzzing, maybe we could just do a LIKE query?